beacon_node/beacon_chain/test_harness/src/validator_harness/local_signer.rs

```diff
@@ -10,7 +10,7 @@ pub struct LocalSigner {
 }
 
 impl LocalSigner {
-    /// Produce a new TestSigner with signing enabled by default.
+    /// Propose a new TestSigner with signing enabled by default.
```

Comment:
I think this is a false-positive.

For best practice I think we should make this:

"Instantiate a new TestSigner with signing enabled by default."